pyvista/cells/cells.py

```diff
@@ -0,0 +1,823 @@
+"""Contains a variety of cells to serve as examples."""
```

Comment:
If these are examples, do we really need to pull them into the main pyvista namespace automatically?

Reply:
I agree, I don't like having example code in the main `pyvista` namespace -- that is what the `examples` subpackage should be used for.

However, this is in line with some of the other geometry methods we have like `Cube()` and `Sphere()` or more sophisticated ones like `Wavelet()`.

I wouldn't be opposed to moving all of these and other's like `Cube()` to the `examples` subpackage, but it may create a migration nightmare

Reply:
I don't have a _strong_ opinion about this, but in my mind Cube/Sphere etc. are different from these new cells.

1. the former are well-defined geometric objects (see also `pyvista/utilities/geometric_objects.py`)
2. the former have a bunch of parameters that can be used to customize them, which means they can be useful in a bunch of real use cases other than examples or debugging
3. the latter smell very "internals" to me: special cases of unstructured grid cells, something very "VTK". Very abstract. Somewhat arbitrary.

Not sure if the above makes sense to others, but I'd be perfectly fine with keeping the cell example stuff in examples (or at least unimported into the main namespace) and still keeping the geometric objects.

Reply:
I'd like to push for these to be in the `examples` namespace -- primarily so that they do not further crowd the `pyvista` root namespace with is already pretty crowded